### ISSUE TITLE
Default primary color should be red

### ISSUE THREAD

**Issue by devnewton** [mblode/marx#57]
**Describe the bug**
A real communist CSS should use more red. Blue is a conservative / capitalist / bourgeois color.

**To Reproduce**
1. Start a proletarian revolution in your country
2. Use lot of red flags
3. Betray workers and destroy your economy with massive corruption and military expense 
4. Wait for people using blue or other non communist color to protest

**Expected behavior**
Red should be used for buttons and links.

**Comment by mblode** [comment]
Fixed in v4.0.0. You can do the following:
```
<link  href="https://unpkg.com/marx-css/css/marx.min.css" rel="stylesheet">
<style>
:root {
  --primary: #da0a0a;
  --primary-400: #d63737;
  --primary-600: #be1918;
  --accent: #faf333;
}
</style>

**Comment by devnewton** [comment]
Thank you !

![image](https://user-images.githubusercontent.com/1859457/146552956-4d71d20c-d78a-4037-b469-9a2806635cf6.png)
